src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -242,6 +309,7 @@ const PermissionManagementDialog = ({
                   text={{ id: 'button.confirm' }}
                   type="submit"
                   style={{ width: styles.wideButton }}
+                  disabled={!userHasPermission || !isValid}
```

Comment:
This form is not validating! At all!

Confirm should not be enabled if I didn't changed the user's permissions.

![bug-permissions-form-not-validating](https://user-images.githubusercontent.com/1193222/107047276-4d241180-67d0-11eb-8d9e-2d1c981d4138.gif)

Not how if I send the transaction, it won't actually create one since the contracts validate this as not being an actual change, so when you get redirected, you run into an error.

We should prevent this by validating the form properly.